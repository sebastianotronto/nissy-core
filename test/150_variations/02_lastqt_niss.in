lastqt
U B2 L (F R')
