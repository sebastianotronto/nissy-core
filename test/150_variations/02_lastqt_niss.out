4
U B2 L (F R)
U B2 L (F R')
U B2 L' (F R)
U B2 L' (F R')
